Tests/XCUITests/HistoryTests.swift

```diff
@@ -140,7 +138,9 @@ class HistoryTests: BaseTestCase {
         // On private mode, the closed tab on regular mode is listed in "Recently Closed Tabs List" as well
         navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
         navigator.performAction(Action.OpenNewTabFromTabTray)
-        closeKeyboard()
+        if !iPad() {
```

Comment:
Yes I also notice that but only in the test (I check on regular build and the keyboard is opening) I double check some of this changes on iPhone and they were passing too but it will be notice to check for both iPhone and iPad to be safe

Reply:
👀 I am checking if the changes breaks the tests for `iPhone 14`.